### PR TITLE
feat(misc) Add OG and Twitter metadata

### LINKED ISSUE #1984: Add twitter card metadata
I think we could easily introduce some metadata to display better content when docs are shared on twitter, nothing big but a nice to have I guess.

### PR DESCRIPTION
**Pull request by EugeneHlushko**
* twitter card
* opengraph

Fixes #1984

### CODE REVIEW

File: template.ejs [hunk 1]
```diff
@@ -6,8 +6,19 @@
     <meta name="description" content="<%= htmlWebpackPlugin.options.context.description %>">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="theme-color" content="#2B3A42">
-    <link rel="shortcut icon" href="/assets/favicon.ico">
 
+    <meta property="og:site_name" content="Webpack" />
```
Comment (montogeek) [inline template.ejs]: `webpack`

**Comment by montogeek** [comment]
Please target `rebuild`

**Comment by EugeneHlushko** [comment]
@montogeek why rebuild?

**Comment by montogeek** [comment]
@EugeneHlushko because https://github.com/webpack/webpack.js.org/pull/2325#issuecomment-401840130

**Comment by EugeneHlushko** [comment]
ok its on `rebuild` now @montogeek

File: src/server.jsx [hunk 2]
```diff
@@ -32,7 +34,16 @@ export default locals => {
...
+          <meta name="description" content={ description } />
+          <meta property="og:site_name" content="webpack" />
+          <meta property="og:type" content="website" />
+          <meta property="og:title" content={ title } />
+          <meta property="og:description" name="description" content={ description } />
+          <meta property="og:image" content={ Logo } />
+          <meta property="twitter:card" content="summary" />
+          <meta property="twitter:site" content="@webpack" />
+          <meta property="twitter:creator" content="@webpack" />
+          <meta property="twitter:domain" content="https://webpack.js.org/" />
```
Comment (montogeek) [inline src/server.jsx]: I am 99% sure that when sharing the link server will return static HTML.
I mean, there is not need for a React component for this.

Comment (EugeneHlushko) [reply src/server.jsx]: i am not sure what you mean, which react component?

Comment (montogeek) [reply src/server.jsx]: From the client side.
I mean, it is fine :)

Comment (EugeneHlushko) [reply src/server.jsx]: ah i thought it was change request, didnt see if thats approved or not :D

**Comment by montogeek** [comment]
Thank you!

**Pull request merged**